backend/BackendMIIT/src/main/java/com/example/BackendMIIT/controller/PassPointController.java

```diff
@@ -0,0 +1,24 @@
+package com.example.BackendMIIT.controller;
+
+import com.example.BackendMIIT.service.PassPointService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(path = "api/miit/points")
+public class PassPointController {
+
+    @Autowired
```

Comment:
Why did you use constructor in DirectionController, but you use `@Autowired` here? I think, we should code in the same style

Reply:
I'm lazy to write the constructor

Reply:
The fast way to add constructor: press `alt`+`insert`, select **Constructor** and select the necessary fields to be added to the constructor